iphone/Classes/TiUISearchBarProxy.m

```diff
@@ -184,9 +184,52 @@ - (id)barColor
 
 - (NSNumber *)style
 {
-  return NUMINT([[self searchBar] searchBarStyle]);
+  return @([[self searchBar] searchBarStyle]);
 }
 
+#if IS_SDK_IOS_13
+- (void)insertTokenAtIndex:(id)params
+{
+  ENSURE_ARG_COUNT(params, 2);
+
+  if (![TiUtils isIOSVersionOrGreater:@"13.0"]) {
+    return;
+  }
+
+  NSDictionary<NSString *, NSString *> *token = params[0];
+  int index = [TiUtils intValue:params[1]];
+
+  UISearchToken *searchToken = [UISearchToken tokenWithIcon:[TiUtils toImage:token[@"image"] proxy:self]
+                                                       text:[TiUtils stringValue:@"text" properties:token]];
+
+  if (token[@"identifier"] == nil) {
+    NSLog(@"[WARN] Missing search token identifier! Using a generated UUID …");
+  }
+
+  searchToken.representedObject = [TiUtils stringValue:@"identifier" properties:token def:[TiUtils createUUID]];
+
+  [[[self searchBar] searchTextField] insertToken:searchToken atIndex:index];
+}
+
+- (void)removeTokenAtIndex:(id)index
+{
+  ENSURE_SINGLE_ARG(index, NSNumber);
+  [[[self searchBar] searchTextField] removeTokenAtIndex:[TiUtils intValue:index]];
+}
+
+- (NSArray<NSDictionary<NSString *, NSString *> *> *)tokens
+{
+  NSArray<UISearchToken *> *tokens = [[[self searchBar] searchTextField] tokens];
```

Comment:
Same  as above. Will crash for iOS < 13. Apply check.
```suggestion
  if (![TiUtils isIOSVersionOrGreater:@"13.0"]) {
     return;
   }
  NSArray<UISearchToken *> *tokens = [[[self searchBar] searchTextField] tokens];
```

Reply:
Same as above - shouldn't be necessary